api/src/test/scala/org/virtuslab/ideprobe/TestScopeParsingTest.scala

```diff
@@ -0,0 +1,49 @@
+package org.virtuslab.ideprobe
+
+import com.typesafe.config.ConfigFactory
+import org.junit.{Assert, Test}
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.virtuslab.ideprobe.protocol.TestScope
+import pureconfig.ConfigSource
+
+@RunWith(classOf[JUnit4])
+final class TestScopeParsingTest {
+
+  private def testParsing(confString: String): TestScope = {
+    val conf = ConfigFactory.parseString(confString)
```

Comment:
We have wrapped this api in `Config` class. It would be better to use the wrapper, in case we ever replaced the underlying implemenation.
This would be `Config.fromString(...).as[TestScope]`